lib/LANraragi/Model/Config.pm

```diff
@@ -101,7 +101,7 @@ sub get_password {
 
 sub get_tagblacklist {
     return &get_redis_conf( "blacklist",
-"already uploaded, forbidden content, translated, russian, chinese, portuguese, french, spanish, italian, vietnamese, german, indonesian"
+"already uploaded, forbidden content, incomplete, ongoing, complete, various, digital, decensored, translated, russian, chinese, portuguese, french, spanish, italian, vietnamese, german, indonesian"
```

Comment:
I don't want to add too much stuff in the blacklist defaults as it shouldn't withhold potentially wanted info from the user.  
That's subjective of course, but I think `decensored` should stay, even if that means having both it and its uncensored variant.